stdlib/2/__builtin__.pyi

```diff
@@ -29,6 +29,7 @@ class classmethod: pass  # Special, only valid as a decorator.
 class object:
     __doc__ = ...  # type: Optional[str]
     __class__ = ...  # type: type
+    __slots__ = ...  # type: Optional[Iterable[Union[str, unicode]]]
```

Comment:
I did not expect that it would work the way it does :D Python 3 works in same manner, so I changed them both into Unions. Tnx!